contracts/zero-ex/contracts/src/transformers/bridges/BridgeProtocols.sol

```diff
@@ -27,29 +27,30 @@ library BridgeProtocols {
     // A incrementally increasing, append-only list of protocol IDs.
     // We don't use an enum so solidity doesn't throw when we pass in a
     // new protocol ID that hasn't been rolled up yet.
-    uint128 internal constant UNKNOWN     = 0;
-    uint128 internal constant CURVE       = 1;
-    uint128 internal constant UNISWAPV2   = 2;
-    uint128 internal constant UNISWAP     = 3;
-    uint128 internal constant BALANCER    = 4;
-    uint128 internal constant KYBER       = 5;
-    uint128 internal constant MOONISWAP   = 6;
-    uint128 internal constant MSTABLE     = 7;
-    uint128 internal constant OASIS       = 8;
-    uint128 internal constant SHELL       = 9;
-    uint128 internal constant DODO        = 10;
-    uint128 internal constant DODOV2      = 11;
-    uint128 internal constant CRYPTOCOM   = 12;
-    uint128 internal constant BANCOR      = 13;
-    uint128 internal constant COFIX       = 14;
-    uint128 internal constant NERVE       = 15;
-    uint128 internal constant MAKERPSM    = 16;
-    uint128 internal constant BALANCERV2  = 17;
-    uint128 internal constant UNISWAPV3   = 18;
-    uint128 internal constant KYBERDMM    = 19;
-    uint128 internal constant CURVEV2     = 20;
-    uint128 internal constant LIDO        = 21;
-    uint128 internal constant CLIPPER     = 22; // Not used: Clipper is now using PLP interface
-    uint128 internal constant AAVEV2      = 23;
-    uint128 internal constant COMPOUND    = 24;
+    uint128 internal constant UNKNOWN         = 0;
+    uint128 internal constant CURVE           = 1;
+    uint128 internal constant UNISWAPV2       = 2;
+    uint128 internal constant UNISWAP         = 3;
+    uint128 internal constant BALANCER        = 4;
+    uint128 internal constant KYBER           = 5;
+    uint128 internal constant MOONISWAP       = 6;
+    uint128 internal constant MSTABLE         = 7;
+    uint128 internal constant OASIS           = 8;
+    uint128 internal constant SHELL           = 9;
+    uint128 internal constant DODO            = 10;
+    uint128 internal constant DODOV2          = 11;
+    uint128 internal constant CRYPTOCOM       = 12;
+    uint128 internal constant BANCOR          = 13;
+    uint128 internal constant COFIX           = 14;
+    uint128 internal constant NERVE           = 15;
+    uint128 internal constant MAKERPSM        = 16;
+    uint128 internal constant BALANCERV2      = 17;
+    uint128 internal constant UNISWAPV3       = 18;
+    uint128 internal constant KYBERDMM        = 19;
+    uint128 internal constant CURVEV2         = 20;
+    uint128 internal constant LIDO            = 21;
+    uint128 internal constant CLIPPER         = 22; // Not used: Clipper is now using PLP interface
+    uint128 internal constant AAVEV2          = 23;
+    uint128 internal constant COMPOUND        = 24;
+    uint128 internal constant BALANCERV2BATCH = 25;
```

Comment:
New Protocol ID. `BALANCERV2` is still used by BeethovenX so couldn't upgrade that in place.